### PR TITLE
[Cloud Security] Fix ECS import method

### PR DESCRIPTION
**Pull request by kfirpeled**
## What does this PR do?

This PR fixes the way `cloud_security_posture` imports ECS mappings. Till now, we did it manually without knowing how to instruct `elastic-package` which ECS version to import its mappings from.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## How to test this PR locally

1. Build locally the package
1. Install version 1.2.10-next
2. Check mappings are as before

Tested with **elastic-package 0.72** (requires **go 1.19** when built locally)

## Related issues

-

### CODE REVIEW

File: packages/cloud_security_posture/data_stream/findings/fields/ecs.yml [hunk 1]
```diff
@@ -1,213 +1,128 @@
...
 - name: host.ip
   external: ecs
-  type: ip
 - name: host.mac
   external: ecs
-  type: keyword
 - name: host.name
   external: ecs
-  type: keyword
-- name: host.os.codename
```
Comment (kfirpeled) [inline packages/cloud_security_posture/data_stream/findings/fields/ecs.yml:91]: `host.os.codename` and `host.containerized` are not part of ECS atm
https://github.com/elastic/ecs/pull/294
https://github.com/elastic/ecs/pull/1512

Do you think it's worth adding them as a custom field? and for the sake of backward compatibility?

Comment (P1llus) [reply packages/cloud_security_posture/data_stream/findings/fields/ecs.yml:91]: As long as the field type does not change between the type you set manually and the type the ECS field ends up being its fine to map them. If its unsure then we should get confirmation about the preferred type first.

File: packages/cloud_security_posture/_dev/build/build.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@v8.3.0
```
Comment (P1llus) [inline packages/cloud_security_posture/_dev/build/build.yml:3]: Should be 8.6.0 if we want to focus on the newest release.

**Review by P1llus — COMMENTED** [review]
Added some small comments

**Review by amirbenun — APPROVED** [review]
Nice

**Pull request merged**